modules/core/src/main/java/org/apache/ignite/internal/processors/cache/GridCachePartitionExchangeManager.java

```diff
@@ -2705,6 +2731,31 @@ else if (fut.isDone() && fut.topologyVersion().compareTo(from) >= 0)
         return true;
     }
 
+    /**
+     * @param blocked {@code True} If take into account only cache operations blocked PME.
+     * @return Gets execution duration for current partition map exchange in milliseconds. {@code 0} If there is no
+     * running PME or {@code blocked} was set to {@code true} and current PME don't block cache operations.
+     */
+    private long getCurrentPMEDuration(boolean blocked) {
+        GridDhtPartitionsExchangeFuture fut = lastTopologyFuture();
+
+        return fut == null ? 0 : fut.getCurrentPMEDuration(blocked);
+    }
+
+    /**
+     * @return Histogram of PME durations metric.
+     */
+    public HistogramMetric getDurationHistogram() {
+        return durationHistogram;
+    }
+
+    /**
+     * @return Histogram of blocking PME durations metric.
+     */
+    public HistogramMetric getBlockingDurationHistogram() {
```

Comment:
According to the code style, we should use `blockingDurationHistogram` instead of `getBlockingDurationHistogram`

Reply:
Thanks. Done.